src/cargo/ops/cargo_run.rs

```diff
@@ -36,7 +36,7 @@ pub fn run(
                 options.filter.target_run(a)
             }
         })
-        .map(|bin| bin.name())
+        .map(|bin| (bin.name(), bin.kind()))
```

Comment:
Would it be better with a struct here, or will a tuple do?

Reply:
tuple is fine!